addons/mail_bot/static/src/owl/components/notification_request/notification_request.scss

```diff
@@ -0,0 +1,109 @@
+// ------------------------------------------------------------------
+// Layout
+// ------------------------------------------------------------------
+
+.o_NotificationRequest {
+    display: flex;
+    align-items: center;
+}
+
+.o_NotificationRequest_content {
+    display: flex;
+    flex-flow: column;
+    flex: 1 1 auto;
+    align-self: flex-start;
+    min-width: 0;
+    margin: 3px;
+}
+
+.o_NotificationRequest_core {
+    display: flex;
+}
+
+.o_NotificationRequest_coreItem {
+    margin: 0 3px;
+
+    &:first-child {
+        margin-left: 0px;
+    }
+
+    &:last-child {
+        margin-right: 0px;
+    }
+}
+
+.o_NotificationRequest_header {
+    display: flex;
+    margin-bottom: 5px;
+}
+
+.o_NotificationRequest_image {
+    width: 100%;
+    height: 100%;
+}
+
+.o_NotificationRequest_imageContainer {
+    position: relative;
+    width: 40px;
+    height: 40px;
+}
+
+.o_NotificationRequest_inlineText {
+    white-space: nowrap;
+    overflow: hidden;
+    text-overflow: ellipsis;
+}
+
+.o_NotificationRequest_name {
+    overflow: hidden;
+    text-overflow: ellipsis;
+
+    &.o-mobile {
+        font-size: 1.1em;
+    }
+}
+
+.o_NotificationRequest_partnerImStatusIcon {
+    @include o-position-absolute($bottom: 0, $right: 0);
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
+
+.o_NotificationRequest_sidebar {
+    margin: 3px;
+}
+
+// ------------------------------------------------------------------
+// Style
+// ------------------------------------------------------------------
+
+.o_NotificationRequest {
+    background-color: white;
+    cursor: pointer;
+    user-select: none;
+
+    .o_NotificationRequest_partnerImStatusIcon {
+        color: white;
+    }
```

Comment:
flatten (no hierarchy), this should be a top-level css rule, so that it has lowest specificity

Reply:
It's a copy/paste of thread preview.